### ISSUE TITLE
Shamans and Cloth Armor

### ISSUE THREAD

**Issue by Fluffalupagus** [dalaranwow/dalaran-wow#5667]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Playing as an Elemental Shaman, at level 24 currently, I cannot select "Need" on cloth items that would be an upgrade for me. I'm almost certain I could select "Need" on retail during WotLK, and would appreciate being able to on this server. Otherwise, elemental shaman seems pointless and I can't imagine what resto shamans do.

**Expected behaviour**: Tell us what should happen instead

I should be able to select "Need" on cloth items

**Steps to reproduce the problem**:

1.  Enter dungeon
2.  Kill boss, drops cloth item
3.  Can only "Greed" or "Disenchant" if 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by galgnu** [comment]
I assume you are supposed to use leather. At lvl 40+ you should use mail. Otherwise the competion for casters would be to hard. Thats why it is tjis way. Pretty sure it was the same in retail but its a long time ago....

**Comment by Fluffalupagus** [comment]
I'm certain it wasnt in retail. It is now, in BfA, but I didnt really play
after cata. I played an ele/resto shaman and the only way to hit the hit
cap was filling a few slots with leather/cloth at 80, and resto absolutely
required cloth/leather to keep up with other healers. Some of the mail
caster items just aren't geared for else shaman until you get into the sets.

**Comment by Golganeth** [comment]
According to the sources I found, this is an intended behavior in WotLK retail. If you provide proof that says otherwise then we can take a look.
http://wowwiki.wikia.com/wiki/Need_Before_Greed

Closing

**Comment by Fluffalupagus** [comment]
In the link provided, you provided proof otherwise for me, albeit a little
fuzzy in semantics. I don't believe I actually played WotLK after 3.2.2, so
my memory isn't entirely accurate.

/quote
At patch 3.3.0 the system was changed so Need Before Greed will now
recognize gear appropriate for a class in three ways: the class must be
able to equip the item, pure melee will be unable to roll on spell power
items, and classes are limited to their dominant armor type (ex. paladins
for plate). All items will still be available via Greed rolls as well as
the new Disenchant option should no member be able to use the item. Also,
players will be able to roll on items with a required minimum level higher
than then player's current level.

The *changes at patch 3.3.0* deliberately introduced a new complaint: *classes
may not roll on gear of lesser armor level even if it is an upgrade, such
as a holy paladin wanting mail with spellpower and intellect.* The only way
for that player to be sure of getting that item is for either for all other
players to pass, or for whoever wins that item to trade it afterwards
provided it wasn't disenchanted. Despite the griping, this is important to
ensure equality with other classes: mail wearers can't roll need on plate,
so plate wearers can't roll need on mail. Blizzard expects people should be
able to find all the gear they need in their own armor class without
poaching from other armor classes.
/quote

If that's the way the server runs, seeing as it runs 3.3.5a that's
understandable. Seems I should have done some more research before running
my mouth! Unfortunately, my memory was correct just a little pre-dated lol.
Seems I'll just be doing some minor arguing in dungeons lol.
